cfgov/v1/tests/test_util_migrations.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import mock
+from unittest import TestCase
+
+from django.apps import apps
+
+from wagtail.wagtailcore.models import Page, PageRevision
+from wagtail.tests.testapp.models import StreamPage
+
+from v1.util.migrations import (
+    migrate_page_types_and_fields,
+    migrate_stream_field,
+    get_stream_data,
+    set_stream_data,
+    is_page,
+)
+from v1.tests.wagtail_pages.helpers import save_new_page
+
+
+class MigrationsUtilTestCase(TestCase):
+
+    def setUp(self):
+        self.root = Page.objects.get(slug='cfgov')
+        self.page = StreamPage(title="Test Page", slug="testpage")
+        save_new_page(self.page, self.root)
+        set_stream_data(self.page, 'body', [
+            {'type': 'text', 'value': 'some text'}
+        ])
+
+        self.revision = self.page.save_revision()
+        self.page.save()
+
+    def tearDown(self):
+        self.revision.delete()
+        self.page.delete()
```

Comment:
It might be simpler to use Django's Test Case (`django.test.TestCase`) instead of `unittest.TestCase` here, as it gives you this database cleanup [automatically](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#transactiontestcase). Doing this protects you if you accidentally forget a `delete` statement when adding to the DB during a test.